iocBoot/iocBPMRFFEApp/st.cmd

```diff
@@ -14,6 +14,9 @@ epicsEnvSet("R", "${EPICS_PV_DEVICE_PREFIX}RFFE")
 epicsEnvSet("STREAM_PROTOCOL_PATH", "$(TOP)/BPMRFFEApp/Db")
 epicsEnvSet("PORT", "BPMRFFE")
 
+asSetFilename("$(TOP)/BPMRFFEApp/Db/accessSecurityFile.acf")
```

Comment:
Should that be installed? I've never quite understood when we install databases and other related configuration and when we do not.

IMHO, we should be able to `make install` to another location and still be able to run it successfully.

Reply:
> IMHO, we should be able to make install to another location and still be able to run it successfully.

Agreed, but I don't know how common/supported that actually is. Most `.db` or `.templates` are installed, but `.proto` files often aren't, and I don't know about `.acf` ones. And even if you look at something like autosave and asyn: the template for autosave records isn't installed, the one for the asyn record is.

I will check if the build system takes `.acf` files in the install list.